examples/datasource-http-backend/package.json

```diff
@@ -19,8 +17,6 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@babel/core": "^7.21.4",
-    "@grafana/e2e": "10.0.3",
-    "@grafana/e2e-selectors": "10.0.3",
```

Comment:
**Just a question:** should we keep these scripts but using playwright now, so they match with what we have [in our templates](https://github.com/grafana/plugin-tools/blob/main/packages/create-plugin/templates/common/_package.json#L13)? I guess probably not super important, as we don't have any meaningful tests, only in a case if someone would check out this folder and start out a plugin from it.

Reply:
Do you mean the `e2e` script? Probably fine to remove I guess - the [migration guide](https://grafana.com/developers/plugin-tools/e2e-test-a-plugin/migrate-from-grafana-e2e#3-update-scripts) has instructions on what scripts to add.